### ISSUE TITLE
'ml commands dogs-cats' failed on Azure Deep Learning Virtual Machine

### ISSUE THREAD

**Issue by simonzhaoms** [mlhubber/mlhub#13]
Terminal outputs of model **dogs-cats** (mlhubber/mlmodels@e135869717edb692220b3f87b9cddbec04125f28):

```bash
$ ml install dogs-cats
...
$ ml configure dogs-cats
...
$ ml commands dogs-cats
Traceback (most recent call last):
  File "/home/simon/Projects/mlhub/virtualenv/dogs-cats/bin/ml", line 11, in <module>
    sys.exit(main())
  File "/home/simon/Projects/mlhub/virtualenv/dogs-cats/lib/python3.6/site-packages/mlhub/__init__.py", line 249, in main
    args.func(args)
  File "/home/simon/Projects/mlhub/virtualenv/dogs-cats/lib/python3.6/site-packages/mlhub/commands.py", line 362, in list_model_commands
    title = re.sub("\.$", "", info['meta']['title'])
KeyError: 'title'
```

**clothes-recommender** (mlhubber/mlmodels@5b42b189c4c0eac01912e1ed9f75c436ca055ea7) has the same problem as well.

**Comment by gjwgit** [comment]
I was able to replicate this on a clean DLVM. Seems to be a problem with the package's DESCRIPTION.yaml file having "description" rather than "title". This was a change a little while ago. I have fixed the DESCRIPTION file for the model. It now works.

Task: capture the error and print a better error message for the user - something like "Malformed DESCRIPTION.yaml file: missing meta -> title."

**Comment by gjwgit** [comment]
Fixed: bda5e0e8a592bec48c54b1c743708d70533eff7b
